src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapCell;
+  width: calendarWidth($cellSize, $daysInRow, $paddingCalendar, $gapCell);
+  padding: $transform $paddingCalendar 0;
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} {
+      :first-child {
```

Comment:
maybe add a day class to the selector?

Reply:
Yeah, U're right) IDK why I did it  this way)